docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -8,6 +8,102 @@ ifdef::notBuildingForSite[]
 include::{includedir}/menu.adoc[]
 endif::notBuildingForSite[]
 
+== Getting Ehcache
+
+The latest version of Ehcache 3.10.0 has been released to maven central under the following coordinates:
+
+=== From Maven
+
+Add the following to the `<dependencies>` section in your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.ehcache</groupId>
+  <artifactId>ehcache</artifactId>
+  <version>3.10.0</version>
+</dependency>
+----
+
+=== From Gradle
+
+Add the following to the dependencies section of your `build.gradle`:
+
+[source,groovy]
+----
+dependencies {
+  implementation 'org.ehcache:ehcache:3.10.0'
+}
+----
+
+=== JAXB Dependencies
+
+Ehcache uses JAXB to parse its XML configuration files. For out-of-the-box functionality it depends on the JAXB
+reference implementation using a version range: `org.glassfish.jaxb:jaxb-runtime:[2.2,3)`.
+If you require a repeatable build you will need to depend on an explicit version within this range.
+If you wish to use an alternate implementation of JAXB then you may need to manually exclude this dependency.
+
+NOTE: If you are using Java 8, or Java 9 (with `--add-modules java.xml.bind`) then you may safely exclude this
+dependency in favor of the implementation provided by the JRE.
+
+=== Jakarta Variants
+
+Ehcache 3.10 adds support for the Jakarta EE namespaced versions of the JAXB APIs.
+If you have moved to the new Jakarta EE namespaced APIs then you may want to choose to use the matching Ehcache variants.
+
+==== From Maven
+
+Add the following to the `<dependencies>` section in your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.ehcache</groupId>
+  <artifactId>ehcache</artifactId>
+  <version>3.10.0</version>
+  <classifier>jakarta</classifier>
+</dependency>
+----
+
+[WARNING]
+====
+The Maven POM cannot capture the dependencies of the Jakarta variant.
+You will also need to manually add the appropriate dependencies:
+[source,xml]
+----
+<dependencies>
+  <dependency>
+    <groupId>javax.cache</groupId>
+    <artifactId>cache-api</artifactId>
+    <version>1.1.0</version>
+  </dependency>
+  <dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-api</artifactId>
+    <version>1.7.25</version>
+  </dependency>
+  <dependency>
+    <groupId>org.glassfish.jaxb</groupId>
+    <artifactId>jaxb-runtime</artifactId>
+    <version>[3,4)</version>
+  </dependency>
+</dependencies>
+----
+====
+
+==== From Gradle
+
+Add the following to the dependencies section of your `build.gradle`:
```

Comment:
Hmmm ... without some discussion of the difference between Gradle and Maven dependency expression here_, you will likely get people unnecessarily adding the above dependencies in a build.gradle...

The mention of differences might flow better if the Gradle version is presented first followed by the Maven version with a sentence as to why the extra stuff is needed.

Reply:
Fixed as discussed.